ext/reflection/tests/ReflectionParameter_getClassName.phpt

```diff
@@ -0,0 +1,25 @@
+--TEST--
+Test ReflectionParameter::getClassName() usage.
+--FILE--
+<?php
+
+use Bar\Baz;
+
+class Foo {
+  public function bar(Qux $qux, $bar, Baz $baz, \Bar\Quz $quz) {}
```

Comment:
Just for the sake of completeness it might be a good idea to add params with `array` and `callable` typehints in the test case even though they aren't strictly needed for this patch's functionality. This would help ensure consistent behavior going forward (e.g. if someone gets a wild hair to change how the "pseudo" hints are implemented you don't want this functionality to break).